src/main/java/net/greghaines/jesque/worker/WorkerPoolImpl.java

```diff
@@ -557,7 +556,16 @@ protected void recoverFromException(final String curQueue, final Exception ex) {
         final RecoveryStrategy recoveryStrategy = this.exceptionHandlerRef.get().onException(this, ex, curQueue);
         switch (recoveryStrategy) {
         case RECONNECT:
-            LOG.info("Ignoring RECONNECT strategy in response to exception because this is a pool", ex);
+            if (ex instanceof JedisNoScriptException) {
+                LOG.info("got JedisNoScriptException while reconnecting, reloading redis scripts");
+                loadRedisScripts();
+            } else {
+                LOG.info("waiting for pool to reconnect to redis", ex);
```

Comment:
Same. Also, consider adding the duration of the wait to the message.

Reply:
done